docs/relnotes.md

```diff
@@ -14,6 +14,7 @@ This is the list of changes to Taipy releases as they were published.
 **`taipy-core`** *<br/>2.1
 
    - New data node named `MongoCollectionDataNode`. It represents the data from a MongoDB collection.
+   - Data node caching mechanism now also takes into account the date and time of the last modification of a file caused by either Taipy execution or an external factor. This behavior is currently limited to CSV, Excel, JSON and Pickle files only.
```

Comment:
'Pickle' should not have a capital letter, just like I wrote in my previous comment.